### PR TITLE
YLP-707 Fix colors in generated PDF report

### PR DESCRIPTION
**Pull request by franck-fourel**
css variables are not interpreted in the PDF report. 
As we will soon rework this report, I applied a quick workaround by removing all the variables from the color definition.

### CODE REVIEW

File: packages/viz/test/styles/colors.test.js [hunk 1]
```diff
@@ -8,18 +8,31 @@ describe('colors', () => {
...
-    expect(_.toLower(colors.veryLow)).to.equal('var(--bg-very-low)');
-    expect(_.toLower(colors.low)).to.equal('var(--bg-low)');
-    expect(_.toLower(colors.target)).to.equal('var(--bg-target)');
-    expect(_.toLower(colors.high)).to.equal('var(--bg-high)');
-    expect(_.toLower(colors.veryHigh)).to.equal('var(--bg-very-high)');
+    expect(_.toLower(colors.veryLow)).to.equal('#DA3A1B');
+    expect(_.toLower(colors.low)).to.equal('#E98D7C');
+    expect(_.toLower(colors.target)).to.equal('#9FCC93');
+    expect(_.toLower(colors.high)).to.equal('#F9D83E');
+    expect(_.toLower(colors.veryHigh)).to.equal('##FFA700');
```
Comment (yvon-dblg) [inline packages/viz/test/styles/colors.test.js:21]:
```suggestion
    expect(_.toLower(colors.veryHigh)).to.equal('#FFA700');
```

**Review by yvon-dblg — CHANGES_REQUESTED** [review]
This fix works, but the tests are failing

```patch
diff --git a/packages/viz/test/styles/colors.test.js b/packages/viz/test/styles/colors.test.js
index 38e24af15..a611c0fc5 100644
--- a/packages/viz/test/styles/colors.test.js
+++ b/packages/viz/test/styles/colors.test.js
@@ -1,38 +1,39 @@
-import colors from '../../src/styles/colors.css';
 import _ from 'lodash';
+import { expect } from 'chai';
+import colors from '../../src/styles/colors.css';
 
 describe('colors', () => {
   it('should export all required colors', () => {
-    expect(_.toLower(colors.bolus)).to.equal('var(--bolus)');
+    expect(_.toLower(colors.bolus)).to.equal('#6fc3bb');
     expect(_.toLower(colors.bolusMeal)).to.equal('var(--bolus-meal)');
     expect(_.toLower(colors.bolusMicro)).to.equal('var(--bolus-micro)');
     expect(_.toLower(colors.bolusManual)).to.equal('var(--bolus-manual)');
     expect(_.toLower(colors.basal)).to.equal('#19a0d7');
-    expect(_.toLower(colors.basalHeader)).to.equal('#DCF1F9');
-    expect(_.toLower(colors.basalAutomated)).to.equal('#00D3E6');
+    expect(_.toUpper(colors.basalHeader)).to.equal('#DCF1F9');
+    expect(_.toUpper(colors.basalAutomated)).to.equal('#00D3E6');
     expect(_.toLower(colors.basalManual)).to.equal('#a8c8d4');
     expect(_.toLower(colors.statDark)).to.equal('#27385b');
     expect(_.toLower(colors.statDefault)).to.equal('#727375');
     expect(_.toLower(colors.statDisabled)).to.equal('#e7e9ee');
-    expect(_.toLower(colors.veryLow)).to.equal('#DA3A1B');
-    expect(_.toLower(colors.low)).to.equal('#E98D7C');
-    expect(_.toLower(colors.target)).to.equal('#9FCC93');
-    expect(_.toLower(colors.high)).to.equal('#F9D83E');
-    expect(_.toLower(colors.veryHigh)).to.equal('##FFA700');
+    expect(_.toUpper(colors.veryLow)).to.equal('#DA3A1B');
+    expect(_.toUpper(colors.low)).to.equal('#E98D7C');
+    expect(_.toUpper(colors.target)).to.equal('#9FCC93');
+    expect(_.toUpper(colors.high)).to.equal('#F9D83E');
+    expect(_.toUpper(colors.veryHigh)).to.equal('#FFA700');
     expect(_.toLower(colors.insulin)).to.equal('#0096d1');
     expect(_.toLower(colors.white)).to.equal('#ffffff');
     expect(_.toLower(colors.axis)).to.equal('#e7e9ee');
     expect(_.toLower(colors.muted)).to.equal('#c1c9d6');
     expect(_.toLower(colors.rescuecarbs)).to.equal('#fa9494');
     expect(_.toLower(colors.deviceEvent)).to.equal('#727375');
-    expect(_.toLower(colors.physicalActivity)).to.equal('#00D3E6');
+    expect(_.toUpper(colors.physicalActivity)).to.equal('#00D3E6');
     expect(_.toLower(colors.confidentialMode)).to.equal('#b4b4b4');
-    expect(_.toLower(colors.smbg)).to.equal('#6480FB');
-    expect(_.toLower(colors.smbgHeader)).to.equal('#E8ECFE');
-    expect(_.toLower(colors.siteChange)).to.equal('#FCD144');
-    expect(_.toLower(colors.grey)).to.equal('#6D6D6D');
-    expect(_.toLower(colors.lightGrey)).to.equal('#979797');
-    expect(_.toLower(colors.darkGrey)).to.equal('#4E4E4F');
+    expect(_.toUpper(colors.smbg)).to.equal('#6480FB');
+    expect(_.toUpper(colors.smbgHeader)).to.equal('#E8ECFE');
+    expect(_.toUpper(colors.siteChange)).to.equal('#FCD144');
+    expect(_.toUpper(colors.grey)).to.equal('#6D6D6D');
+    expect(_.toUpper(colors.lightGrey)).to.equal('#979797');
+    expect(_.toUpper(colors.darkGrey)).to.equal('#4E4E4F');
 
   });
 });
```

File: packages/viz/test/styles/colors.test.js (hunk 1 above)
Comment (franck-fourel) [reply packages/viz/test/styles/colors.test.js:21]: let me fix the tests, I did not looked at the toLower/toUpper and the way color codes are written

**Review by yvon-dblg — APPROVED** [review]
LGTM

**Pull request merged**